Week2/js-exercises/ex2-whatsYourMondayWorth.js

```diff
@@ -14,6 +14,14 @@
 
 function dayWorth(tasks, hourlyRate) {
   // put your code in here, the function does returns a euro formatted string
+  const totalTime = tasks
+  .map(item => item.duration)
+  .map(item => item/60)
+  .map(item => item * hourlyRate)
+  .reduce((a,b) => a + b,0)
+  return totalTime;
+  
+
```

Comment:
It says the output of the function should be a euro formatted string. this is not I believe